### DISCUSSION TITLE
Losing credential with the update

### DISCUSSION THREAD

**Discussion by funkr** [iterate-ch/cyberduck#14935] [Q&A]
I update today to the latest version (8.6.0) and when the new version came up it has lost my s3 bucket credentials.
Not so cool!

**Comment by tatze96** [comment]
Cyberduck updates don't touch your saved credentials as they are kept within the _Windows Credential Manager_, or respectively, _Keychain Access.app_.

From which version did you update to 8.6.0? 

The behavior might occur if you updated from a version prior to 6.4.0, but otherwise, it is rather unlikely that the update process removed the credentials.
